packages/design-tokens/css/variables.css

```diff
@@ -0,0 +1 @@
ading-3-font-weight: 700; --typography-heading-3-font-size: 1.375rem; --typography-heading-3-line-height: 1.875rem; --typography-heading-3-letter-spacing: normal; --typography-heading-4-font-family: "Inter", "Noto Sans", Helvetica, Arial, sans-serif; --typography-heading-4-font-weight: 600; --typography-heading-4-font-size: 1.125rem; --typography-heading-4-line-height: 1.5rem; --typography-heading-4-letter-spacing: normal; --typography-heading-5-font-family: "Inter", "Noto Sans", Helvetica, Arial, sans-serif; --typography-heading-5-font-weight: 600; --typography-heading-5-font-size: 1rem; --typography-heading-5-line-height: 1.5rem; --typography-heading-5-letter-spacing: normal; --typography-heading-6-font-family: "Inter", "Noto Sans", Helvetica, Arial, sans-serif; --typography-heading-6-font-weight: 700; --typography-heading-6-font-size: 0.875rem; --typography-heading-6-line-height: 1.5rem; --typography-heading-6-letter-spacing: normal; --typography-paragraph-intro-lede-font-family: "Inter", "Noto Sans", Helvetica, Arial, sans-serif; --typography-paragraph-intro-lede-font-weight: 400; --typography-paragraph-intro-lede-font-size: 1.25rem; --typography-paragraph-intro-lede-line-height: 1.875rem; --typography-paragraph-intro-lede-letter-spacing: 0; --typography-paragraph-intro-lede-max-width: 975px; --typography-paragraph-body-font-family: "Inter", "Noto Sans", Helvetica, Arial, sans-serif; --typography-paragraph-body-font-weight: 400; --typography-paragraph-body-font-size: 1rem; --typography-paragraph-body-line-height: 1.5rem; --typography-paragraph-body-letter-spacing: normal; --typography-paragraph-body-max-width: 780px; --typography-paragraph-small-font-family: "Inter", "Noto Sans", Helvetica, Arial, sans-serif; --typography-paragraph-small-font-weight: 400; --typography-paragraph-small-font-size: 0.875rem; --typography-paragraph-small-line-height: 1.125rem; --typography-paragraph-small-letter-spacing: normal; --typography-paragraph-small-max-width: 680px; --typography-paragraph-extra-small-font-family: "Inter", "Noto Sans", Helvetica, Arial, sans-serif; --typography-paragraph-extra-small-font-weight: 400; --typography-paragraph-extra-small-font-size: 0.75rem; --typography-paragraph-extra-small-line-height: 1.125rem; --typography-paragraph-extra-small-letter-spacing: normal; --typography-paragraph-extra-small-max-width: 600px; --typography-paragraph-bold-font-weight: 600; --typography-button-primary-font-family: "Inter", "Noto Sans", Helvetica, Arial, sans-serif; --typography-button-primary-font-weight: 700; --typography-button-primary-font-size: 1.125rem; --typography-button-primary-line-height: 1.5rem; --typography-button-primary-letter-spacing: normal; --typography-button-secondary-font-family: "Inter", "Noto Sans", Helvetica, Arial, sans-serif; --typography-button-secondary-font-weight: 500; --typography-button-secondary-font-size: 1rem; --typography-button-secondary-line-height: 1.5rem; --typography-button-secondary-letter-spacing: normal;}
```

Comment:
Whoah. Is this a generated file we can update easily? Or a source file that we maintain (we'd want more line breaks if so)?

Reply:
It's generated via the build command!

Reply:
Would it be easy to add a comment at the top to say it's auto generated? If not don't worry about it :)

Reply:
Ah can do, done.